Source/PLATEAURuntime/Public/PLATEAUImportSettings.h

```diff
@@ -103,6 +135,14 @@ class PLATEAURUNTIME_API UPLATEAUImportSettings : public UObject {
         };
     }
 
+    static TMap<EPLATEAUMeshGranularity, FText> GetGranularityTexts() {
+        TMap<EPLATEAUMeshGranularity, FText> Items;
+        Items.Add(EPLATEAUMeshGranularity::PerPrimaryFeatureObject, LOCTEXT("PrimaryFeatureObject", "主要地物単位"));
+        Items.Add(EPLATEAUMeshGranularity::PerAtomicFeatureObject, LOCTEXT("AtomicFeatureObject", "最小地物単位"));
```

Comment:
対応逆になっているので修正お願いします。
```suggestion
        Items.Add(EPLATEAUMeshGranularity::PerPrimaryFeatureObject, LOCTEXT("PrimaryFeatureObject", "最小地物単位"));
        Items.Add(EPLATEAUMeshGranularity::PerAtomicFeatureObject, LOCTEXT("AtomicFeatureObject", "主要地物単位"));
```